codechefcli/decorators.py

```diff
@@ -32,3 +32,53 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def sort_it(func):
+    '''
+    desc: decorator method to sort the specified argument
+    '''
+    def wrapper(*args, **kwargs):
+        sort = args[0]
+        if args[1]:
```

Comment:
You can store `args[1]` in a variable to improve readability, like you did with `args[0]`